src/beam.cpp

```diff
@@ -260,7 +260,7 @@ void BeamSegment::CalcBeamInit(
     // Calculate the extreme values
 
     int yMax = 0, yMin = 0;
-    int nbRests = 0;
+    size_t nbRests = 0;
```

Comment:
Just in case there will be more than [2147483647](https://en.wikipedia.org/wiki/2,147,483,647) rests... 😜

Reply:
Yes, why changing this? We use `int` everywhere in verovio and static cast to it when necessary (e.g. in `.size()`). This never cause any problem and is not likely to cause some.

Reply:
Yes, why changing this? We use `int` everywhere in verovio and static cast to in when necessary (e.g. in `.size()`). This never cause any problem and is not likely to cause some.

Reply:
Personally I think this would be more readable, but this one was actually connected to another thing, that has been fixed. Reverted it with #1720.